postprocessing/janus-pp-rec.c

```diff
@@ -131,6 +132,12 @@ int main(int argc, char *argv[])
 			ignore_first_packets = val;
 		JANUS_LOG(LOG_INFO, "Ignoring first packets: %d\n", ignore_first_packets);
 	}
+	if(g_getenv("JANUS_RTP_HEADER_EXT_ID_CVO") != NULL) {
```

Comment:
The convention we followed for other env variables, is to use the prefix `JANUS_PPREC_`. As such, I'd rename it to something like `JANUS_PPREC_EXT_CVO` or `JANUS_PPREC_EXTID_CVO`, which might come in handy if we decide to parse other extensions in the future.